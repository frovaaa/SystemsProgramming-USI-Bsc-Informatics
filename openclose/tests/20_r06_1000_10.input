bbbdbbppqbdppbbddpqbpqdqbdbpbppppbpbppqppbdpqbdppbbdppbdpppbbbbbpbpppqqqdbppbdbbpppppbdbpbbpqdbd4pbdqppbdqbpqbddqbbddpbdbdbp7ppqbpqbdpqpqbddbdbdpbppppqbpbdpppbbbbddbdpbdpbbddbdpppbdbpqbbpbpqbdbbbpbbpbbpp5pqbpbbbdbdddqbbbdbdppbdbpqdbpbdpqpbbbpppppqqbdbdpqqqqppbppqpbbbdbdbpqppqbbdbbbdpqbddbbbpbpbppqppbb8bpbdbpbppppppbdbbpqppbbpqpppppqqbbddqpppqpbbpbdpppbppbpbbbpbbbbdbpbbpppqpbbdpqpppbpbpbpbdbbdpbbbppbbpbdbbdppbpbbdbpqdbpbpqbdbbpqpqdbdbbddpbbbddbdbppqppqbbbddpbdpppqbpppbdbpqdbdpqqpbppbpppbpqpbpbdpqqppqbppqqdqbpbppbdbbdpbdpppqbpqdbbbbpqppppbbppb6bpppbbddbdqpbbddpbbpqdbbbdpqdddbdbpbpqdqbpqdppppqqbdpqbbdbpqpbdbdqbpbppbdpqqqdbdpbdbdqbbpqpqdpbdppqbbbpbbddqpbdqdbddbddbppbdqbdqppbbpbdpbbbdpbdqbpqpppbbdppbdbbbdpbpqbpqddpqqpqddqqdqqqdddqqddqqdqqdqdddqqdqqqqddqqddqqqqddddqqqdqqdqddqdqqqdqqdqqqdqqqdqqddqddqddqdqqdqddqqdddqdqdqdqdqqqdqqqddqddddqdddqdqqdqqqqddqqqqqddqqddqqqqqqdqdqdddqqqdqdqdddddqdddqqdqqdddqqdqqdddqqddqddqddddqdddqqqqqddqqqqdqqqdqqqdqdqdqqqqqddqqdqdddddqqqqqdqqqqqdqdqqqqdqdqqdddd
pbpppqqpbdqbppppqbppbpqbdpqpqppqbdqp5bpqpqdppbbpppppbbdpppbppqpbdbdqbppqpbpqpbpqpqdpbbppbd2bpq9bbbbppqqppqqpqbp6pbpbdqbbbbbpppbdppqppbbppbbddbpbbpqpbdbdppbdqqpqbbbbpqbpbbbbbdbbpbbbdppbbpbbdbpbdpbdpbdqbbdpqpqbddpqpqbbbdddbpqpbdpbpqdqqdpqqbbpbdbbppbpqdqbbdbpqbpbbddpbpqppbpqdbbdbdbpbpqdppbbdbbdbdpbpppqqpbbbdbppppqqqbdbpbdqpbdpppqbdqqqdpbdbbbddpqdbdbdqqpbbbpbdbpbbbbbdpbbpbbppqppqbpqpbbbddpbdqppbbdpqbdbdbdppbdqqdqbpqbdbdbpppbpqppbbddqbbbddpqbpbpqdbpbdbpbdppbpqdppqbppqbpqbpbpbdbpbbpqppqqddqpbpqbpqdbbdbbdbpqbddbpqppbpqpqdbdbbpqpbppqbbbdpppbpbdbbdbbpqpqpbbbpppqpqbbbpbpbbddpppqq2bdpbbbbpbdbpqbpbdqbpqdpqddbdpqpqbdqbppqqd1pppqbpqdqbbpqbpbdbpqpqdbppqbbdbddpqppbdbbbddbbdbbdbdbdpqbdbbbdpbppbpppqpqqbdpppppbppqbpqpppbpqpbdbdqppqpbpbpbdqbbpqdddpqpqqbbbdpqbdddbddqqdqqqdqdqqqqqqdqqdqdddddqqqdqdddqddddqqqdqdddqqdddqdddqdqqqddqdqddqqddddqdqdqddqdqqqqdqdqddqdqqqddqdqdqqddqddqddddqdqdddqdddqqdqddqqqddqqdqqdddqddqddqddqddqqddqddddddqdddddqddqdqqddqqqqqqddddddqqddddddqqddqqdqqdqdqqqdqqqqqddqqqdqqdqqqdqdq
bdbdpbpbppqbdqdqpqbdpppqqbbpqbdppbpqpqbdpbpqbdppqbbdbpbbdbddqpbbddqdbppqqbpbppqbbbbdbpbbbbpqbdpbpppbpbdppbbdbbppbpbbbdbpbpqbpbpbdqdpqbdqbdbbdpqpbdbdbpqpbbbbbbpqbbpbbpbdqdpqdp9pbdbdppbbbbbpbbpqpppbdppqbdqbpqdqqbbppbdppbpbpqdbbpppqpqqpbdbbddqbbbppqppbbpppqpppqbdqqbpppbppbpqdpqbdqbdqbddqppqpqqqppqbdqqdpqpbbdpqpqpbbpqdpbbbdbpqbbddbdbbbbbdpqpqpbdqbpqbpbdbpbdqbbpbbpqbpppbdpbdbdqpqbbddqbpbdbdqbdbpqddbbpqdpbdbbpbbdpbdppqqqdpbdbbbbpppbpqpqbdbbppqppqbdbpbppqpqqpbbbppbppppbdbp7bbdbpppbppbbppbdqbbppppqpqqqqpbpbpqdpbpbpqbbddppqppbpbpqdqdqqqbppqqddbdqpbbdbdbddbpqdbdbbpbdpbppqqppbbbbpqpqbdbddpbbpbdbpppbpqdpppqppbbddqpppbbpqbbpqdbpbbpqdbdpbbbbbbpqbpqddppppqbpqdbdpppbbpbdqbpqppbpppqqppbpppqpqbbpbdqppqbbpppbpqdpbbbbdbbpqdpbdbpqpppbbbbbpppbppbpbppqqdqdqqdqqqdddddqqqdqddddqqqqddqddqqdqqqdqqdddqqqqqqdddddqdqddddqqqqqqqqqdqddqdddqqdqqddqdqqdqddqddqqdqqqddqdqqqqdqqdddqdqdqqdddqqqddddqqddqdqqdddqdddqdddddddddqdqdqqqddqqqdddqddqdqqqqddqddqdddddqqqqqddddddddqdqdddqdddqdqqdddqqqdqqqdqddddqddddqdqdddqdqdqqddqdq
pbbbpbpqbddbbbdbbdbpqbbpqpqbbddbpqdpbpppqpbbppbbpqbpbdbdbpqdqpbpppqppppbdbpqdqpqbbddpqppqbbdbppqbdpqpbdbpqppqpppqbpbdbpqbdpqbbdddbpqdpbpppbpbpqpbpbpbpqdppqbbbpqbbbddpqpqdpqbdpqpqppbppqbdbdqppbpppqbpqppqpqbdbbpqpqbpbbbbbdpbdpbdqqb0bpbpppppqqppqpqbpbbbddppqpqqdbpbbpbbpqbbpbpbdqdqdpqbbbbppqbppbdpbbddqqb4bppqbdqdppbbddqpqpbbpbdppppbppbppqbbdpqbdpbbppqqdppqppbbddbppbbbddpp5bdbpqbddbdppqqqpbpppbppbpbdqbdbpqdppqqbddqqpqdppqppqbdppbbbdbddppqpbbbpppqbppqqpbdpb5pbpqbdppqpqqbdppbdqbpqpbdqdbdbdppbbd8ppbbppqppbdbdqpbbdpbpbpqpqdpbbbbdppbdbdqpqbpbdbpbppqbdbdbpbpqbpbpqdqbddpbdpbbdpqppqppqpbbbbdbdddpbpqpbpbpqppqqpqbddqdqbpbdb40ppbbd7bbpbdppqpbdqqbppbppbpbdqpbpbpqdbppbpbbpqbbdbbdbdbbddppqbdbpqdqbddppbbbbddpbbpbpbppqbpbbdbddbppqbdqbddbbddbdpbdbpqbpbbpbppbdbbbdddqqdqddqddqqdqdqdqddqddqqdddqdqqdqdqdqqdqqdqdddqqdqddqdqqqdqqdqdqdqdqdqdddqqdqdqqqddqqdqqqdqdqqdqqdddqqdqqqqqqqdqqdqqdqqqdqdqdqqdqqqqqddqqdqdqdddddddqddqdqdqqqqdqddddddqdddqdqqdqqdqqdddqqdqdqdqdqqqdqqdqqqdqqddqqqqqqdqdddqqddqqqdqddddddqdddq
pbpbpbdpbdqpqqbddqpppbbbbpbbppbdqbdbbppbppbbbbdbdpbbbpbbpppqbpbdqdqqdbpqddbbpqdbbdbdddqbbbppqpqqpqbpbdqpppqpbbpbpqdbdqdbddbdqqbpbpppqqppqqqppbdpqpqqbdqbbbpbbddpqpqpqppqpppppqpbpbpqdbppbbbddpqbbdbbdbbpqbdpqbdbppqbdqddpbpppppbbdbbbdbbdpbpqppbdqpqqpbppqbbbbb7bdbbdbdpbbpp4ppbdqpbdppqqqpqbdbpppqqpbpbpppqbpbbpbdqbddpbpqbbdbdbdpbdppp1bpqdbppqppqqqpbpbbppqqbbbdpbbppbdqbdbpqdbbpqbbdbddpqdbbddbbdbdpbbbddbpqpbbdbpbdqdppqbbbdbpqdbdddbdppppqqbbpqd2dbpbdqbpqddqpppqqpbbddpqbdpqbbbpbbddbdqpqpbbppbppqqdpbpqdqpbdqqqpqd1pbppqbbdpbbpqbbpppbbpqddpbbdppqbdqpbppqpqqdqbbdddbdqbdbdpbbbdddbdbppbbdbdbbbpqbbbpqbbpqbbbpbppqpqqpbpqpbdbppppbppqqpppqppqqpqqqdqbd3bppqpppbdbpqbpqpbpbdppqbdbpbbd7bbpbdpppbdqpbbbbpbdqdpbppbppbbbbbdpbpqbpqpbppbdbbbpbbbbbbpqbpqpp2pqbdbdpppqpbpqdpqppppqqbdqbdbbbdddqqqqqqdddddddqdddqqdqddqddddqqdqqdqdddqqqqdddqdqqdqddqqqqdqqqdqdqdqddddddddddddqqdqqqqddddqdqdqdqdddqqqqdqddqddqddqddddqdqdqqqqddqdqdqqdqdqqdqqqddqddddddqdqdqddddqqqqqdqddddqqdqdqqqqqqqddddqdqdddddddqdddqqdqqddqddqddddqqqdq
pbbdbbdbppqbbpbdppqqpbdppqpbpbpbpbdp5bbbdpbbdbbppqbdpppbbbppbbpbbdbbddpbbddpqqdqppbdbdbpqppqqppqqppbpqdbdppqpqppqpppbbbddpbdpbdbpqdqpqbdbpbdpqpbdqbbpbdpbppqbdqpqbddbbpbbdd1pqbdbbbbddddqpbdbdbpppqppppqqpbdqqbppbbdpbbbbbddpqdpbdqppqbppppqbdbpqdqpb4dbbbbddppqbdqpqbpbbpbdqpqpqbppppqbpbppqpqpqpbbpqbpbdqdbddpqppqqbbpqbbdddpqpqppqqpqdbbdbdppbpqppqqdqpbdbpbbdppqppppqbpbpqpbdqpppqqpqbpbpqdqppqppbbpbbdbbpqdpbpbdpbpqdqqpqpppppbdqpbbppbdbpbbddqpbpqpqp4pbdppqpqbpppqpqbbdpqdqbpbdbpbdqdbdpqp9bpppbbbpbdpqpqqdbbdpbbpbbdpbbpqddpbbdpbpqdpqpbbbpbbpqd7pbppbpbdbbpbdqbd5pbpqbpppbdpqpbdbdbdbdbpbbpppqpbdbdbbbpbpbpbbbb1pqpbdbdpbpbpbppbbpppbbppbppbppqpqpqqpbdbdqdbbppqqdbddbdqpqpqbdbdbdbpqdqpbbddqdpppppqbpppppqqqbdbppbdbppbbpqbpqpqppppbpqdbdqbbbppqqpbppppqqbdqpqqdqppqqppbdpqqppqqqdddqqqdddqqdqqdqqdqqqqqqddqqqddqqdqdqdqqddddqdqdqdddqqqddqdqqqqddqddqdqqdqdqdddqqdqqdqddqdddqqqdqqdqdqqqdqdqqddqqqqqdqddqddqqqdqdqdqqqqdqdqqddqqdqdqqqdddqdddqqqdqddqdqqdqqqdqddqqddqdqdqqqqqqqdqqddqqdddqqqqdddqddqqdqdqdqqqqddqdddq
bpbdbdbdbpppbbbddbbpbdpqpbpqbpbbbpqbpppbppbppqbbdpqbbbddbdpqbbddpbpbdqdqbpbdqbbbbddpbdpbpbdbpqppbbpqbbbdddppqpqqdpqbbbddpbbppppqppqppbdbdqpqbpqpqpppbppppqqpqbbdpqppbdpqpqqbpqdqdbbpbpbdqbddbpppppqqbbdbppbdppqqqpbbp1bbpqbbpbdqdpbppbpppqpb4bd7ppqpbdbpppqppbdpbbbppbbpbpbppppqbbpqppbpppbppqqbbddpqdpbpqdqqq6pbbbpqpqbd3pbbdbdbdbdppppbpqbdbpqdpbbbbdppqbbbddbpppbbbpqbdppqbpqdbbddpbbpqddqbpqbdbdbpbdppbppbpqdpppppppqbbddpqpppqbpqdbdqqbpbp5bppqpbpqppp2bppqpbdbpqbpb6bdpqbbpbpbbbpbdbdqbpqbpbbppbdqbbdpbpbpqpppqqqbddbbbpbppbpbdbppbdbpppbpqbddqpqqpppppbbpqddbdbdbdpqqbdqbbddqqbbpbpqdbdqdbppqqdppqppqpqqpbbpqppbppbdbppbbpbpqpqdpbbddqbbppbpbpqppbpqbbdppqppbdqqbpqdpbdqppbppqpbbpbpbpqdqbbbpqbbbppqbdbbddqdpqdpbdbdqdppppqpbpqbdpqdqpqbdqbpq9ppqqbdpqpqpqdbpqpbbdbddqdqqddddqddqqdqqqddqqdqdqqddqddqqdqqdqqddqqdqqdqqdqdqqdqdddqdqdqddqdddddqdqdddqddqqdqqqdqqdqdqdqqqqqqqqdqqqddqdddqqqddqdddqdqqqqdqdddqqdqqddqqqdqdqddqqdddqqqqqdqqdqqqdqqdqdddqddqqddqqqdqddqqdqqqdqqqqqddqddqqdqdqqdddddqdqqdqqqddddqddqqdddqqqdqd
ppbbpqpbdbdqbbbbpqbppqqbbpqdppbdbdqpppbdpqppbbddpppbbbppqqbpbbbbdpbdpqpbdppbdpbppbdbbbpqdpqddbbbpbpbbpbdbpqdbpbpbdqbp22pbbpppbbpbpbppbpbpbpbpbbdbddpqpbbpqpbbdpppbbpqppqpbbbbdppqqpbdqdppqbdqpqppppqbpqbddbbppppbpbdqbdpqdqbdqbpqdbdqqpqpqbbbppqqdddpbppbpbbbpqpqppqbpbbdpqdbdbpqpbbppqbdpqpbpbbbbdbdddppqpbbdbbppbpqbpqdbdpqdbdqqbdddbbbbpbdbpbdpbpqbbbppppqqqbbpqddpbdppbpbbpppbppqqpppppbbddpppbdqbpbbpppbppbpqbbddbdpbdqbdpqdqqdqbpbbdpqdbbppqpbdbbbdpqbbpbdqdbbppbpqdbpqb4pqbpqdbdbdpbbppbpbbbbppbbpbppbpbbpbdqddpppppqbdqbdqqpppqbdbdqbppppbpqbpqdbpppqppqqqqddbppqqdbpppqbpbbdbbdpbdqbbdppqbpbpbbppppppqqqbpqdpqpbbbpppqqqbbppbbbpqdbppqppbdqpqbdpq6bbbdppppqppbdqqbpqdbdpbbppqpqpbpbbdbdbbbbdppbppqbdppppbbpbdppqpbbdbppbdqqbbpqbppqqpppqqppbdbbppqqddqqqdddddqqqddqqqqqdqqddddqdqqddqqqqddqqdddqqdddddqqqqddqdqdqdddqdqqdqqqqdqqqdqqdqddqqddddqdqqddqddqqdddddqqddqdqqddqdqqqqqqqdqqqddqdqqqqddddqqdqdddddqqdqdqqddqdqdqdddqdqqdqddqqqddqqddqqqdqddqqdqdqdqdqqdqdqddqqqddqqddqdqddqdqdddqqdqqqqqdddqddddqqqqqqqqqddddddddqq
pbb9pbbpqdppqbbdbpbbbppbdqppqpbbpqbdbbbbbpbdqbpqbbdpqpbppqqdqdbbbpbbbpqdddppqppbbdbpbpbpqppqqpqpbppbbbbpbpbdpqbpppqbbdpbpbbpqdpbbdbbdpqbdpbdppqppbpbpqdpbbpqddpqbdbbdbpqbpbppqqdbpbdpbdbpqpbbbdpqpbpbdqpppbppqbpqpqbdbppbpqpppqppqqpqpbppqqdqpbbdbdbbpbdbbdbbbbdppqpqbbbbpbdqpqbpbbddbppqppqqppqpppbbpppbdpqpbbpbppqpbdpqqqppqbbdpppqbbbpqddpqpbpqpqbpqbbddpbdqbbbdbbppbbpbdbdbdpbbbpqdpbbdppbdqpqqpqpbpbbpqbpqpqpqbpqbpqbppbbbbdpqdbddbdpbpbbbbppppbdqpqppppqbpppqbbddqqbbdbbddbbdppbbdbdbpbdqddqqdbbddpqbddpqbpbpbbddqbdpbbpbpqpppbpqbbpqppqqdpbbdpqppppqpbdbdpbdbpbdqbbbdbbbpbppqpbdqpbbbdppqpbdbbpqpbpqdpqbdqppqbdqpbbdbpbdqpqppqpppbbdpqbpbdpbdpppqqqppqbppbdqbbbdpqbbpqbbpqd3pbdqbppbdpqqqddbdbbbpbdppbbppbbbbdpbpqpqdqpqbbppqqpqdpqpbdbpqdbdqdpbdqppbdpqbpbdqpqpppqbbdpqbbdddqqdqqdddqqddqqqdddddddqdqqqddqqqqddqddqqddqqdqddddddqqqqqdqddqqqdqddqdqddqqqqqqddddqdqdqqddddddqdqdqddqqddqqddddddqdqqdqdqddqqqqddqqqqqdqdddddqddddqdddqqqdqqddqdqqqdqddqdqqdqdddqqdqqqqddqdqdqdqqdqdqddddqqdqdqdqddqqqqdddddddddddqqqdddqddqdqddq
bdbpppqpbppppppbbppqpqqppq5pbpbdppppbdpqbbpqbbdppqqdbdbdbbpqddpbbdbpqpqpbbpqbdpqddqbbpqbbbbbpqbddpqdpbdbpbpppqqqdbdbdqpqdbpqbbbppqqbpppbbppppbbdbbbbpbpbbpqbbbpbbpbbbpqpbdqbpqpbppbpbdbdpbpbdqdbbpqdbdbpbdpbp8ppbpbdppbpqppbppbppppbppbdqqbppqpbbbbddppbpppppbdqbpqdppbpqbpbppbdpppqpbdbdppqbppbpqdqbpqdqdpbpqbdpbpbbdbppbpbpppbdbdbpbdpqbdbbdpqdqpbbpbdqppbdqqppqqddqdpbbpbbpbbpqpqpbdbdbpbdbpbdqpppqppbbbdbbdbpqpppqpppbdbpbpbpppbbbbbbbdbpbdpbppqqbdbpbdqbbdpppbdpqbpbdqdbpbbbddpqpqppppbbbbppqqbpbbddqdbbbbbpppbdppqqpqbbpqbbdbdddbbppqppqbppbdbdqbdppbdpbbpppqqpqqbdbbdpbbpbpppqqbbbdbbpqdddbpqddppbbpppqbdpqbbddbdqpqqdbdpbbpqpqddqpqppppbbpqbpbbpbdqppppppqqqpqppbdpqbpbdbbpbbddpbppbbbpbpppbpbppqpqpqpbdqqbpqppqbbpbdbbpqbddbbdbbddpppbbbbddddqqqddqddqddqdqqqdqdddqqdqqddqdqqqqqddqdddqqqqdqqqdqddqdddqqqqdqqdddqqqdddddddddqqqqdqdqqqdddqqdddddddqqqdqdqdqqqqqddddqqqqdqdqddqddqddqqqqdqdqqddqdqdqqqqqqqdqddqqqqqqdqqddqqddqqqqdqqdqqdqqqdqqqdqqddqqdqqdqddddqddqdddddqdqdddddqqqqddqqqdddddqdddddddqddqqqqqdqqddqqqqqqdqqqd
