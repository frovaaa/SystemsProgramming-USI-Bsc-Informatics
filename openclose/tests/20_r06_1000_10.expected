bbbbppbpbppppbpbpppppbpppppbbbbbpbppbbpppppbpb4pbp7ppbpppbppppbbpppppbbbpbbbbpbbpbbpp5bbbppbppbbbppbppbbbpbbbbbpbpbpppbb8bpbpbppppppbbppbbpppppbbppppbppbpbbbpbbbbpbbpppbpppbpbpbpbpbbbppbbpbppbpbbpbbpbbppbpppbpppbppbpppbpbbpbppbpppbbbbppppbbppb6dqqddqqqqddddqqqdqqdqddqdqqqdqqdqqqdqqqdqqddqddqddqdqqdqddqqdddqdqdqdqdqqqdqqqddqddddqdddqdqqdqqqqddqqqqqddqqddqqqqqqdqdqdddqqqdqdqdddddqdddqqdqqdddqqdqqdddqqddqddqddddqdddqqqqqddqqqqdqqqdqqqdqdqdqqqqqddqqdqdddddqqqqqdqqqqqdqdqqqqdqdqqdddd
pbpbpppbppbp5ppbbpppppbpppbpbppbppbbpp2b9bbbbbp6pbbbbbbppppppbbppbpbbpbbbbbpbbbbbbpbbppbbpbbpbbpbbpbbbppbppbbpppbbpbppbbbpbbbpbpbbbbpbbpbbppbpbpbbpppbpbbpbpbppppbpbbpbpbpbbbbppbbpbpbbpppbpbbbpbbbppbbbpbpp2pbbbb1ddddqqqdqdddqqdddqdddqdqqqddqdqddqqddddqdqdqddqdqqqqdqdqddqdqqqddqdqdqqddqddqddddqdqdddqdddqqdqddqqqddqqdqqdddqddqddqddqddqqddqddddddqdddddqddqdqqddqqqqqqddddddqqddddddqqddqqdqqdqdqqqdqqqqqddqqqdqqdqqqdqdq
pbpbbppbpbpbbbpbpbbbbpbbbbpbpppbpppbbbppbpbbbpbbbpbpbbbbbbbbpp9pppbbbbbpbbpbbppppbpbbpbbbpppbbpppbpbpbbbbbbbbbpbbbpbbbppbpbbppbbbbpppbbbppbpbpbbbppbppppbp7qdqqqqdqqdddqdqdqqdddqqqddddqqddqdqqdddqdddqdddddddddqdqdqqqddqqqdddqddqdqqqqddqddqdddddqqqqqddddddddqdqdddqdddqdqqdddqqqdqqqdqddddqddddqdqdddqdqdqqddqdq
pbbbpbbbbbbpbpppbbppbbbpbppppppbbppbpppbppbpppbpbpbpbppbbbppbppbppbpbbbpbbbbb0bpbppppbpbpbbpbbbbbbbpbpb4ppbbpppppbppbpbpbpppbppbpp5qpbpppppppbppbbbppbppb5pbpppb8ppbbpppbpbppbbbpbpbpbpbpbppbpppbpbbpb40ppb7dqqdqddqdqqqdqqdqdqdqdqdqdddqqdqdqqqddqqdqqqdqdqqdqqdddqqdqqqqqqqdqqdqqdqqqdqdqdqqdqqqqqddqqdqdqdddddddqddqdqdqqqqdqddddddqdddqdqqdqqdqqdddqqdqdqdqdqqqdqqdqqqdqqddqqqqqqdqdddqqddqqqdqddddddqdddq
pbpppbbbbpbbpbbppbppbbbpbbbbbbbpbpbbbbpppppppbpbppbbbbpbpppppbbbbpbpbpbbbbb7bpbbpp4pbppbpbppbpbpbbpppp1bpbpbbbbpbbpbbpbbpbpppb2dqppbbbpb1pbpbpbbbbppppbppbbbbbbbbbbbbpbpbpbppp3bppppbbpbppbpb7bbppppbbbpbppbppbbbbpbbpbppbbbpbbbbbbbpp2qqdddddddqdddqqdqddqddddqqdqqdqdddqqqqdddqdqqdqddqqqqdqqqdqdqdqddddddddddddqqdqqqqddddqdqdqdqdddqqqqdqddqddqddqddddqdqdqqqqddqdqdqqdqdqqdqqqddqddddddqdqdqddddqqqqqdqddddqqdqdqqqqqqqddddqdqdddddddqdddqqdqqddqddqddddqqqdq
pbbbpbbppppbpbpbpp5bbpbbbppppbbbppbbppbpppppppbpbpbbppbbp1qpbpppbppbpbbpbpppb4dbbbpbbbpppbpbppbbppbpbppppbpbpbpppbbpbbpbpppppbbppbpbp4ppbpbpp9bpppbbbpbbpbppbppbbbpb7pbppbpbb5pbbppppbpbbpppbbbpbpbpbbbb1ddddqdqdqdddqqqddqdqqqqddqddqdqqdqdqdddqqdqqdqddqdddqqqdqqdqdqqqdqdqqddqqqqqdqddqddqqqdqdqdqqqqdqdqqddqqdqdqqqdddqdddqqqdqddqdqqdqqqdqddqqddqdqdqqqqqqqdqqddqqdddqqqqdddqddqqdqdqdqqqqddqdddq
bpbpppbbbppbbpbbbbpppbppbpbbbbbppbpbppbbpbbpppppbpppbppbbpbpppbbppbbp1bbbpbppbpppb47ppbpppppbbbppbbpbpbpppbbppbp6pbbb3pbppppbpbbbpbbpppbbbpbbpppbppppppppbpbp5bppbppp2bppbbpb6bbpbpbbbbbpbbpbpbpbbbpbppbpbppbppbppbbppbppbppbbpbbppbpbppbbpppbppbbpbbbbppb9dqqddddqddqqdqqqddqqdqdqqddqddqqdqqdqqddqqdqqdqqdqdqqdqdddqdqdqddqdddddqdqdddqddqqdqqqdqqdqdqdqqqqqqqqdqqqddqdddqqqddqdddqdqqqqdqdddqqdqqddqqqdqdqddqqdddqqqqqdqqdqqqdqqdqdddqddqqddqqqdqddqqdqqqdqqqqqddqddqqdqdqqdddddqdqqdqqqddddqddqqdddqqqdqd
ppbbbbbbbbpppppppppbbbbpbbbpppppbppbbbpbpbbpbpbbp22pbbpppbbpbpbppbpbpbpbppbbpbpppbbppbbpppbbpbppbpbbbpbpbpbbppbpbppbbbbbpbppbbbbppppbpbbpppbpppppppbpbbppbpbbppbbbbbppbb4pbbppbpbbbbppbbpbppbpppbppppbppbpbbbpbpbpbbppppbbbbbppbbbpp6qqdddqqdddddqqqqddqdqdqdddqdqqdqqqqdqqqdqqdqddqqddddqdqqddqddqqdddddqqddqdqqddqdqqqqqqqdqqqddqdqqqqddddqqdqdddddqqdqdqqddqdqdqdddqdqqdqddqqqddqqddqqqdqddqqdqdqdqdqqdqdqddqqqddqqddqdqddqdqdddqqdqqqqqdddqddddqqqqqqqqqddddddddqq
pbb9pbpbbpbbbpppbbbbbbbbbbbppppbbpbpbpbppbbbbpbpbppbpbpbpbbppppbppbbbpbppbpbbpbpppbpbbppbpppbbbpbbbbpbbbbbpbpppppbbppppbbpbpbppbpbbbbbbppbbppbbpbpbpbbbbbbppbpbpbbbbppppppbbpbpbbpbpppbbpbpppppbbbbbbpbppbbppbbpbbppppbbpppbpbbbbb3dddddqdqqqddqqqqddqddqqddqqdqddddddqqqqqdqddqqqdqddqdqddqqqqqqddddqdqdqqddddddqdqdqddqqddqqddddddqdqqdqdqddqqqqddqqqqqdqdddddqddddqdddqqqdqqddqdqqqdqddqdqqdqdddqqdqqqqddqdqdqdqqdqdqddddqqdqdqdqddqqqqdddddddddddqqqdddqddqdqddq
bpppbppppppbbp5pbpppppbbpbbbbbbbpbbbbbpppbbppppbbbbbpbpbbbbbpbbpbbbbpbppbppbbppbp8qdqqddqqdqqdqddddqddqdddddqdqdddddqqqqddqqqdddddqdddddddqddqqqqqdqqddqqqqqqdqqqd
